keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -165,6 +170,25 @@ Determine the date that a given secret was last used:
 
 If `kube-apiserver-legacy-service-account-token-tracking` is unavailable, no secret would be removed.
 
+Mark the secrets as invalid and recover:
+
+1. The secrets will be added a label with `kubernetes.io/invalid-legacy-token`: true.
+2. If the users use the invalid tokens, in the Validate() function of
+   "kubernetes/pkg/serviceaccount/legacy.go", it will detect the usage of
+   invalid tokens and return the error information, telling the users to
+   re-activate the token by updating the label value or use the tokenrequest. At
+   the same time, the tokens will be updated with the new `kubernetes.io/legacy-token-last-used` date.
+3. If the users don't use the invalid tokens, one year later after the tokens
+   are marked as invalid, the tokens will be finally deleted.
```

Comment:
we would need to make the `kubernetes.io/invalid-legacy-token` label value the date it was marked invalid, so we can tell when it has been marked invalid for the correct length of time (and I'd probably change the label to `kubernetes.io/legacy-token-invalid-since`)

that would also make it easy for an admin to find tokens where use is being attempted after being marked invalid